spec/features/user_adds_interactions_spec.rb

```diff
@@ -0,0 +1,90 @@
+require "rails_helper"
+
+feature "user adds interactions", %(
+  As an authorized user
+  I want to add interactions to my vocabulary
+  So that I add them to my bots
+
+  Acceptance Criteria
+  [X] I must go to a page to add an interactions
+  [] I will see a different form depending of the category of the interaction
+  [X] I must be redirected to my vocabulary page.
+
+) do
+  let!(:user) { FactoryGirl.create(:user) }
+
+
+  scenario "user goes to the new interaction page clicking on add vocabulary" do
+    login(user)
+    visit user_interactions_path(user)
+    click_link 'Add vocabulary'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.